.pre-commit-config.yaml

```diff
@@ -27,12 +27,17 @@ repos:
         files: ^api/src/.*\.py$
 
   - repo: https://github.com/PyCQA/bandit
-    rev: 1.6.2
+    rev: 1.7.4
     hooks:
       - id: bandit
         args: [-l, --recursive, -x, tests]
         files: ^api/.*\.py$
 
+  - repo: https://github.com/Lucas-C/pre-commit-hooks-safety
+    rev: v1.2.4
+    hooks:
+    -   id: python-safety-dependencies-check
```

Comment:
A shorter name; "safety" for example, would make this easier to run from CLI

Reply:
Done!